### ISSUE TITLE
OSC 52 does not work in splitted panes

### ISSUE THREAD

**Issue by Mic92** [wez/wezterm#836]
## Describe the bug

In panes opened by `wezterm.action{SplitHorizontal={domain="CurrentPaneDomain"}}` the text copied with OSC 52 does not end up in the clipboard. The first original pane works but not the second one.

To reproduce run the following code and try to paste it some where

```
printf '\033]52;c;%s\a' Ki5zcWwKKi50YWdzKgoqLmdlbXRhZ3MqCioudG1wKgoqLm1pbi5qcwoqLm1pbi5jc3MKKi5wbGlzdAoqLnBkZgoqLmEKY29tcGlsZV9jb21tYW5kcy5qc29uCnRhZ3MK
```

The content should look like: 

```
*.sql
*.tags*
*.gemtags*
*.tmp*
*.min.js
*.min.css
*.plist
*.pdf
*.a
compile_commands.json
tags
```

This bug is similar to https://github.com/wez/wezterm/issues/764

## Environment (please complete the following information):

see https://github.com/wez/wezterm/issues/764 
Tested commit d93be9e19ecc0b2674cf00ec91cbb79d32ccb01d

## To Reproduce

see bug description

## Configuration

```lua
local wezterm = require 'wezterm';
return {
      -- ctrl-b ;
      {key=";", mods="LEADER", action=wezterm.action{SplitHorizontal={domain="CurrentPaneDomain"}}},
      {key="b", mods="LEADER|CTRL", action=wezterm.action{SendString="\x02"}},
}
```

## Expected behavior

The text is copied to the clipboard

## Screenshots

-

## Session Recording

-

## Additional context

-

**Comment by wez** [comment]
Heh, although this manifested similarly, this time it was an issue with the client side not connecting the remote tab to the clipboard in some cases!!
This is fixed now in 1a442555298945c0766bb1a5ad4b18b507506efd

**Comment by Mic92** [comment]
Wow this was fast. Thanks!
